squirrel.worker/src/main/java/org/dice_research/squirrel/analyzer/commons/SquirrelTripleHandler.java

```diff
@@ -0,0 +1,111 @@
+package org.dice_research.squirrel.analyzer.commons;
+
+import org.apache.any23.extractor.ExtractionContext;
+import org.apache.any23.writer.TripleHandler;
+import org.apache.any23.writer.TripleHandlerException;
+import org.apache.jena.graph.Node;
+import org.apache.jena.graph.NodeFactory;
+import org.apache.jena.graph.Triple;
+import org.dice_research.squirrel.collect.UriCollector;
+import org.dice_research.squirrel.data.uri.CrawleableUri;
+import org.dice_research.squirrel.sink.Sink;
+import org.eclipse.rdf4j.model.IRI;
+import org.eclipse.rdf4j.model.Resource;
+import org.eclipse.rdf4j.model.Value;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * 
+ * Triple handler for Any23 RDF extraction
+ * 
+ * @author Geraldo de Souza Junior
+ *
+ */
+
+public class SquirrelTripleHandler implements TripleHandler {
```

Comment:
If the handler is only working with Any23, its name should reflect that :wink:

Reply:
Actually, it should be used by any other issue that might use Clerezza